lighthouse-core/runner.js

```diff
@@ -133,14 +133,24 @@ class Runner {
             a => Aggregate.aggregate(a, runResults.auditResults));
         }
 
+        const runtimeConfig = {
+          environment: [
+            {name: 'CPU Throttling', value: !opts.flags.disableCpuThrottling},
```

Comment:
let's put CPU throttling last of the three and add the extra detail hardcoded in here.

> Network throttling (150ms RTT, 1.6Mbps down, 750Kbps up): enabled
> Device emulation (Nexus 5X): enabled
> CPU throttling (5x slowdown): disabled